ExpApproach updates

UlrikeS91 opened the pull request:
All approaches have an InterLex ID now. A few comments/questions for @lzehl and @tgbugs:

1) I couldn't find "chemogenetics". Does this exist yet?

2) "genetics" and "informatics" are listed under "scientific discipline" in InterLex. Will those get a new/updated entry or is this was we should use here?

3) "optogenetics" exists twice (http://uri.interlex.org/ilx_0108079 &  http://uri.interlex.org/ilx_0737732). Definitions were rather similar, so I just picked one. Any objections? 

4) The "preferred ID" was the InterLex ID. So I copied the same one in for the "interlexID" as well. In almost all cases this was the preferred ID according to the InterLex entry. @lzehl if you don't want this repetition, we can remove them from the preferred ID again.

UlrikeS91 committed:
Update anatomy.jsonld

UlrikeS91 committed:
Update behavior.jsonld

UlrikeS91 committed:
Update cellBiology.jsonld

UlrikeS91 committed:
Update cellMorphology.jsonld

UlrikeS91 committed:
Update cellPopulationCharacterization.jsonld

UlrikeS91 committed:
Update cellPopulationImaging.jsonld

UlrikeS91 committed:
Update cellPopulationManipulation.jsonld

UlrikeS91 committed:
Update clinicalResearch.jsonld

UlrikeS91 committed:
Update computationalModeling.jsonld

UlrikeS91 committed:
Update developmentalBiology.jsonld

UlrikeS91 committed:
Update ecology.jsonld

UlrikeS91 committed:
Update electrophysiology.jsonld

UlrikeS91 committed:
Update epidemiology.jsonld

UlrikeS91 committed:
Update epigenomics.jsonld

UlrikeS91 committed:
Update ethology.jsonld

UlrikeS91 committed:
Update evolutionaryBiology.jsonld

UlrikeS91 committed:
Update ethology.jsonld

UlrikeS91 committed:
Update epidemiology.jsonld

UlrikeS91 committed:
Update expression.jsonld

UlrikeS91 committed:
Update expressionCharacterization.jsonld

UlrikeS91 committed:
Update genomics.jsonld

UlrikeS91 committed:
Update histology.jsonld

UlrikeS91 committed:
Update metabolomics.jsonld

UlrikeS91 committed:
Update microscopy.jsonld

UlrikeS91 committed:
Update morphology.jsonld

UlrikeS91 committed:
Update multimodalResearch.jsonld

UlrikeS91 committed:
Update multiomics.jsonld

UlrikeS91 committed:
Update neuralConnectivity.jsonld

UlrikeS91 committed:
Update neuroimaging.jsonld

UlrikeS91 committed:
Update omics.jsonld

UlrikeS91 committed:
Update physiology.jsonld

UlrikeS91 committed:
Update proteomics.jsonld

UlrikeS91 committed:
Update radiology.jsonld

UlrikeS91 committed:
Update spatialTranscriptomics.jsonld

UlrikeS91 committed:
Update transcriptomics.jsonld

UlrikeS91 committed:
Update genetics.jsonld

UlrikeS91 committed:
Update informatics.jsonld

UlrikeS91 committed:
Update genetics.jsonld

UlrikeS91 committed:
Update optogenetics.jsonld

tgbugs commented:
There are a number of issues here that I need to work through here related to the identifiers. I'm not sure whether you should use the existing terms for genetics and optogenetics because their modelling is different and we will need to figure out how to deal with merging them.

I would suggest using the `mod:` as the preferred ids since they are what is used in the ontology and they now resolve! (yay! http://uri.interlex.org/tgbugs/uris/readable/modality/Anatomy).
https://github.com/SciCrunch/NIF-Ontology/blob/methods/ttl/approach.ttl
I have created mod:Genetics and friends but we need to get them interlex ids (@tmsincomb) so that they will resolve. We will update the InterLex records to reflect this now that they resolve correctly.

For reference, check the children tab [here](https://scicrunch.org/scicrunch/interlex/view/ilx_0770148#children) for the list of the actual approach terms that is currently in InterLex.

UlrikeS91 committed:
Update anatomy.jsonld

UlrikeS91 committed:
Update behavior.jsonld

UlrikeS91 committed:
Update cellBiology.jsonld

UlrikeS91 committed:
Update cellMorphology.jsonld

UlrikeS91 committed:
Update cellPopulationCharacterization.jsonld

UlrikeS91 committed:
Update cellPopulationImaging.jsonld

UlrikeS91 committed:
Update cellPopulationManipulation.jsonld

UlrikeS91 committed:
Update clinicalResearch.jsonld

UlrikeS91 committed:
Update computationalModeling.jsonld

UlrikeS91 committed:
Update developmentalBiology.jsonld

UlrikeS91 committed:
Update ecology.jsonld

UlrikeS91 committed:
Update electrophysiology.jsonld

UlrikeS91 committed:
Update epidemiology.jsonld

UlrikeS91 committed:
Update epigenomics.jsonld

UlrikeS91 committed:
Update ethology.jsonld

UlrikeS91 committed:
Update evolutionaryBiology.jsonld

UlrikeS91 committed:
Update expression.jsonld

UlrikeS91 committed:
Update expressionCharacterization.jsonld

UlrikeS91 committed:
Update genetics.jsonld

UlrikeS91 committed:
Update genomics.jsonld

UlrikeS91 committed:
Update histology.jsonld

UlrikeS91 committed:
Update informatics.jsonld

UlrikeS91 committed:
Update metabolomics.jsonld

UlrikeS91 committed:
Update microscopy.jsonld

UlrikeS91 committed:
Update morphology.jsonld

UlrikeS91 committed:
Update multimodalResearch.jsonld

UlrikeS91 committed:
Update multiomics.jsonld

UlrikeS91 committed:
Update neuralConnectivity.jsonld

UlrikeS91 committed:
Update neuroimaging.jsonld

UlrikeS91 committed:
Update omics.jsonld

UlrikeS91 committed:
Update optogenetics.jsonld

UlrikeS91 committed:
Update physiology.jsonld

UlrikeS91 committed:
Update proteomics.jsonld

UlrikeS91 committed:
Update radiology.jsonld

UlrikeS91 committed:
Update spatialTranscriptomics.jsonld

UlrikeS91 committed:
Update transcriptomics.jsonld

UlrikeS91 commented:
Thanks, @tgbugs. I've made some adjustments according to your feedback.

1. I removed interlex ID, preferred ID and synonyms from "chemogenetics", "genetics", "informatics" and "optogenetics". These are the only four that aren't part of the experimental modalities (https://scicrunch.org/scicrunch/interlex/view/ilx_0770148) yet. As soon as they are added and/or get interlex IDs, we can update those again. 
2. For the rest, I changed the preferred ID to "http://uri.interlex.org/tgbugs/uris/readable/modality/<<name>>".

@lzehl, this PR should be ok to merge now (but decide for yourself :P).

lzehl commented on instances/experimentalApproach/electrophysiology.jsonld:
```diff
@@ -6,9 +6,9 @@
   "knowledgeSpaceLink": null,
   "name": "electrophysiology",
-  "preferredOntologyIdentifier": "http://uri.interlex.org/base/ilx_0741202",
+  "preferredOntologyIdentifier": "	http://uri.interlex.org/tgbugs/uris/readable/modality/Electrophysiology",
```
I think there are too many spaces here

UlrikeS91 replied:
Yes, happened while copy-pasting. Fixed it.

lzehl reviewed (changes requested):
except one (I think) typo it looks good.

UlrikeS91 committed:
Update electrophysiology.jsonld

lzehl commented:
@UlrikeS91 also here: could you transfer this please also to the controlledTerms repo? The openMINDS_instances repo will be deprecated. Instances are version dependent and are therefore now stored on the respective version branch of the openMINDS metadata model repo.

UlrikeS91 commented:
> @UlrikeS91 also here: could you transfer this please also to the controlledTerms repo? The openMINDS_instances repo will be deprecated. Instances are version dependent and are therefore now stored on the respective version branch of the openMINDS metadata model repo.

Done :)
